docs/src/guide.md

```diff
@@ -0,0 +1,58 @@
+# Guide for package authors
+
+New packages, or a new version of packages, are added the Julia General registry by pull requests against the [JuliaRegistries/General](https://github.com/JuliaRegistries/General/pulls) GitHub repository. This is automated by the [Registrator.jl](https://github.com/JuliaRegistries/Registrator.jl) GitHub bot and the [web interface](https://github.com/JuliaRegistries/Registrator.jl). 
+
+It can be helpful to also install [TagBot](https://github.com/JuliaRegistries/TagBot), which automatically tags a release on your repository after the package or new version of the package is added to the registry.
+
+The process usually proceeds as follows:
+
+Either
+
+* Install the Registrator bot on the repository corresponding to the package you would like to add or update in the General registry, following the instructions in the readme at [Registrator.jl](https://github.com/JuliaRegistries/Registrator.jl). Then file an issue on the repository, or make a comment on the commit, with "@JuliaRegistrator register". For more information on interacting with the Registrator bot, see the aforementioned README.
+* Go to the [web interface](https://github.com/JuliaRegistries/Registrator.jl) and follow the instructions linked there to start the registration process.
+
+After either option, a pull request is made against the [JuliaRegistries/General](https://github.com/JuliaRegistries/General/pulls) GitHub repository.
+
+Often, that is all one needs to do. New packages are subject to a 3-day waiting period for community comment, while new versions of existing packages may be merged right away. Once the pull request is merged, the package or new version of the package is available in the General registry. If TagBog is installed, then it will tag a release once the pull request is merged. This is very helpful to ensure that tagged releases correspond exactly to versions in the registry.
```

Comment:
```suggestion
Often, that is all one needs to do. New packages are subject to a 3-day waiting period for community comment, while new versions of existing packages may be merged right away. Once the pull request is merged, the package or new version of the package is available in the General registry. If [TagBot](https://github.com/JuliaRegistries/TagBot) is installed, then it will tag a release once the pull request is merged. This is very helpful to ensure that tagged releases correspond exactly to versions in the registry.